README.md

```diff
@@ -63,6 +63,14 @@ tropopause usage is similar to that of troposphere, the principle difference is
 
 ## Available Objects
 
+### tropopause
+* `Tags` - A replacement for troposphere Tags, concatenating tags deduplicates Keys, with precendence to the rightmost expression.
```

Comment:
Markdown lint:

Should be a new line after header items.
Should be a new line before ``` blocks.